tests/test_client_factory.py

```diff
@@ -30,6 +30,14 @@ def test_session_invalid_profile(self, mock_cache_lookup, mock_cache_set):
             cache.session(profile="non-existent-profile")
         self.assertEqual(mock_cache_lookup.called, True)
 
+    @mock.patch("taskcat._client_factory.Boto3Cache._cache_set", autospec=True)
+    @mock.patch("taskcat._client_factory.Boto3Cache._cache_lookup", autospec=True)
+    @mock.patch("taskcat._client_factory.boto3.Session", autospec=True)
+    def test_imported_session(self, mock_boto3, mock_cache_lookup, mock_cache_set):
+        x = Boto3Cache()
+        x.import_session("foobar", mock_boto3, "us-east-1")
+        self.assertEqual(mock_cache_set.called, True)
```

Comment:
We should be asserting that the cache entry has the desired prefix.

mock has a `called_once_with` method that could acheive this easily